### ISSUE TITLE
🐛 Standardize admin create pages

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#8457]
## 🐛 Bug

The admin create pages should follow a template and include the same elements.

- `/admin/settings/skills/create`
- `/admin/settings/skills/families/create`
- `/admin/settings/departments/create`
- `/admin/settings/classifications/create`
- `/admin/teams/create`
- `/admin/pools/create`

## 🦋 Expected Behaviour

Each page should be the same in its use of copy and layout.

## 📋 Steps to Reproduce

1. Navigate to admin create pages
2. Observe different headings

## 📸 Screenshots
![Screen Shot 2023-11-09 at 15 27 40](https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/1438ff9a-c461-450a-a5fe-5d067b32c90b)
![Screen Shot 2023-11-09 at 15 27 53](https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/b3f272bb-3168-40b9-9f05-b678e40d34e8)
![Screen Shot 2023-11-09 at 15 28 04](https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/48523d30-6f33-43f8-93b5-6f42f64d8760)
![Screen Shot 2023-11-09 at 15 28 14](https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/12101053-1403-4de1-8982-c2b38628d567)
![Screen Shot 2023-11-09 at 15 28 24](https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/a349e16b-3a58-4e11-922f-6a7a7e53206b)
![Screen Shot 2023-11-09 at 15 28 34](https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/15181154-10de-4f5f-967e-8af1dbdef403)
![Screen Shot 2023-11-09 at 15 28 42](https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/6053bf71-49a8-478a-8c3d-12ec5b841280)

## 🙋‍♀️ Proposed Solution

Create one template for all to use. Each uses similar copy, actions, layout.

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the bug was addressed.

- [ ] all the admin create pages appear the same

**Comment by mnigh** [comment]
@GCTC-NTGC/designers is this something you are able to address?

**Comment by Jerryescandon** [comment]
Yeah, I think this one is worth tackling. I suggest @gobyrne add an issue on the design backlog and link it to this one.

**Comment by gobyrne** [comment]
Ok, moving this issue over to design backlog. @Jerryescandon it's still work for you to do though ;)
